src/dashboards/components/DashboardContainer.tsx

```diff
@@ -63,7 +122,7 @@ const DashboardContainer: FC<Props> = ({autoRefresh, dashboard}) => {
 
 const mstp = (state: AppState) => {
   const dashboard = state.currentDashboard.id
-  const autoRefresh = state.autoRefresh[dashboard] || AUTOREFRESH_DEFAULT
+  const autoRefresh: any = state.autoRefresh[dashboard] || AUTOREFRESH_DEFAULT
```

Comment:
Setting the type here to any isn’t all that beneficial. Why do it?

Reply:
An explicit type would help resolve some of the confusion I had around the `upper` time comparison in the comment above